modules/rostests/kmtests/tcpip/TcpIp_user.c

```diff
@@ -113,7 +113,7 @@ START_TEST(TcpIpConnect)
     Error = KmtSendToDriver(IOCTL_TEST_CONNECT);
     ok_eq_ulong(Error, ERROR_SUCCESS);
 
-    WaitForSingleObject(AcceptThread, INFINITE);
+    WaitForSingleObject(AcceptThread, 10 * 1000);
```

Comment:
Maybe add a test check whether it timed out:
```suggestion
    Error = WaitForSingleObject(AcceptThread, 10 * 1000);
    ok(Error == WAIT_OBJECT_0, "AcceptThread timed out\n");
```